controllers/controller.js

```diff
@@ -45,3 +43,23 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getComments = (req, res, next) => {
+  const { article_id } = req.params;
+  const articleIdNumber = parseInt(article_id, 10);
+
+  if (articleIdNumber > 9000) {
+    return res.status(404).send({ msg: "ITS OVER 9000!!!" });
+  }
+  if (isNaN(articleIdNumber)) {
+    return res.status(400).send({ msg: "Invalid article_id" });
+  }
```

Comment:
No need for the isNaN one as SQL will give you an error code which you can use to throw the error